external-storage-uri.md

```diff
@@ -54,6 +54,10 @@ The following is an example of an Amazon S3 URI for [`IMPORT INTO`](/sql-stateme
 s3://external/test.csv?access-key=${access-key}&secret-access-key=${secret-access-key}
 ```
 
+> **Note:**
+>
+> Please note that the XML API of GCS is not fully compatible with the `s3://` protocol. If you are using GCS, please use the `gcs://` protocol instead.
```

Comment:
In the first use of GCS in this note we should expand the abbreviation. For example:

> Please note that the XML API of Google Cloud Storage (GCS) is not fully compatible with the `s3://` protocol. If you are using GCS, please use the `gcs://` protocol instead.

Reply:
We should also expand it's first use on this page (line 8 above).